src/styles/01-colors/_colors-brand.css

```diff
@@ -0,0 +1,23 @@
+/** Brand Colors */
+:root {
+    /** What's On is used for Exhibitions & Events. */
+    --section-whats-on-primary: #8a1a9c;
+    --section-whats-on-secondary: #7c1097;
```

Comment:
Not necessarily a question about the PR but about the colors: as a dev, is it acceptable to use any of these colors for anything that's not for these specific pages? In other words, can these also be used as utility colors?

Reply:
My _current_ understanding is no, so i think it would be good to express that in the notes somehow. (NYPL theming is a moving target unfortunately.) I'm also thinking that forward-facing, we'll want to expose these values not as NYPL names but maybe as more generic `--theme-1-primary`, `--theme-1-secondary`, etc. for external apps like SimplyE and Lyrasis. Kinda tempted to bundle that in with this work so we can have one big change all at once, but I don't think it would be breaking to do that in a follow-up. I'll log a ticket